src/wallet/mod.rs

```diff
@@ -1084,7 +1084,7 @@ where
     }
 
     /// Sign a transaction with all the wallet's signers, in the order specified by every signer's
-    /// [`SignerOrdering`]
+    /// [`SignerOrdering`]. Returns true if the PSBT was finalized, and false otherwise.
```

Comment:
nit: This function returns the `Result` type with an encapsulated `bool` that has the value true if the PSBT was finalized, or false otherwise.

Reply:
Updated